pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -250373,7 +250373,6 @@ self: {
        description = "Bindings to ZeroMQ 4.x";
        license = stdenv.lib.licenses.mit;
        hydraPlatforms = stdenv.lib.platforms.none;
-       broken = true;
```

Comment:
Do not edit this file! Note the comment at the very top.

Reply:
Okay, what file should I be editing instead? The comment at the top of the file doesn't have this information.

Reply:
My first guess would be `configuration-hackage2nix.yaml`, but there doesn't seem to be an entry for  `zeromq4-haskell_0_8_0`.